.github/workflows/release-serverless-init.yml

```diff
@@ -55,56 +56,57 @@ jobs:
         run: ./build_serverless_init.sh
         env:
           AGENT_PATH: datadog-agent
-          VERSION: ${{ inputs.tag }}
+          VERSION: ${{ github.event.inputs.tag }}
           SERVERLESS_INIT: true
-          AGENT_VERSION: ${{ inputs.agentVersion }}
+          AGENT_VERSION: ${{ github.event.inputs.agentVersion }}
 
       - name: Set up build directory and copy binaries
         run: |
           mkdir ./scripts/bin
           cp .layers/datadog_extension-amd64/extensions/datadog-agent ./scripts/bin/datadog-agent
           cp .layers/datadog_extension-amd64-alpine/extensions/datadog-agent ./scripts/bin/datadog-agent-alpine
 
-      - name: Login to DockerHub
+      - name: Login to GHCR
         uses: docker/login-action@v2
         with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_TOKEN }}
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
have you already set the secrets? (and in 1password?)

Reply:
it's a per-repo secret set automatically by github, we don't manage it